src/zones/invert.ts

```diff
@@ -0,0 +1,44 @@
+import { normalize } from './normalize';
+
+/**
+ * Convert an array of exclusions and keep only from / to
+ *
+ * The method will always check if from if lower than to and will swap if required.
+ * @param exclusions - array of zones
+ * @param options - options
+ * @param options.from - Specify min value of zones (after inversion)
+ * @param options.to - Specify max value of zones (after inversion)
+ * @returns array of inverted zones
+ */
+
+export function invert(
+  exclusions: { from: number; to: number }[] = [],
+  options: { from?: number; to?: number } = {},
```

Comment:
In order to document the default values (as you did in the other project) you need to add the documentation for the options at this level.